crates/status/src/impl/bitstring_status_list/syntax/entry_set/mod.rs

```diff
@@ -40,13 +58,18 @@ impl BitstringStatusListEntry {
     /// Creates a new bit-string status list entry.
     pub fn new(
         id: Option<UriBuf>,
+        status_size: StatusSize,
         status_purpose: StatusPurpose,
+        status_message: Vec<StatusMessage>,
```

Comment:
nit: why not make this field match the struct field? (i.e. either make both plural or both singular?)